src/gd_kotlin.cpp

```diff
@@ -243,10 +178,10 @@ void GDKotlin::init() {
                     jvm_jmx_port = "9010";
                 }
             }
-        } else if (cmd_arg.find("--jvm-to-engine-shared-buffer-size") >= 0) {
+        } else if (cmd_arg.find("--jvm_to_engine_max_string_size") >= 0) {
             String result;
             if (split_jvm_debug_argument(cmd_arg, result) == OK) {
-                jvm_to_engine_shared_buffer_size = result.to_int();
+                jvm_to_engine_max_string_size = result.to_int();
                 //TODO: Link to documentation
                 LOG_WARNING(
                         vformat("Warning ! Buffer capacity was changed to %s, this is not a recommended practice",
```

Comment:
Warning should be changed or removed IMO

Reply:
While at it the link to the docs should be added: https://godot-kotl.in/en/latest/advanced/commandline-args/